day3/day3_rd.py

```diff
@@ -0,0 +1,109 @@
+from collections import Counter
+from typing import List
+
+
+def read_diagnostic_report(raw_data_file: str) -> List[List[int]]:
+    """
+    Reads raw data into a list of lists of bits
+    """
+    with open(raw_data_file, "r") as f:
+        data = []
+        while line := f.readline().rstrip():
+            data.append(list(map(int, line)))
+    return data
+
+
+def convert_bit_list_to_int(bit_list: List[int]) -> int:
+    """
+    Converts a list of bits (e.g., [1, 0, 0, 1, 1]) to its integer form
```

Comment:
Guess I should've gone with:

```suggestion
    Converts a list of bits (e.g., [1, 0, 0, 1, 1]) to its decimal integer form
```